ratatui-widgets/src/barchart.rs

```diff
@@ -1472,4 +1473,18 @@ mod tests {
         ]);
         assert_eq!(buffer, expected);
     }
+    #[rstest]
```

Comment:
nit: newline before tests

```suggestion

    #[rstest]
```